client/me/reauth-required/index.jsx

```diff
@@ -111,47 +110,45 @@ const ReauthRequired = createReactClass( {
 				code: this.state.code,
 				remember2fa: this.state.remember2fa,
 			},
-			function ( error, data ) {
+			( error, data ) => {
 				this.setState( { validatingCode: false } );
 				if ( error ) {
 					debug( 'There was an error validating that code: ' + JSON.stringify( error ) );
 				} else {
 					debug( 'The code validated!' + JSON.stringify( data ) );
 				}
-			}.bind( this )
+			}
 		);
-	},
-
-	codeRequestTimer: false,
+	};
 
-	allowSMSRequests: function () {
+	allowSMSRequests() {
```

Comment:
Looks like this should be made an arrow function in order to automatically bind `this`?

Reply:
That's true. I decided to inline the entire function body into the `setTimeout` callback that uses it.